Exercise5.3_Week9.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<!--Week 9 Assignment: Wioleta Takuska-->
+<html>
+
+<head>
+    <title>Wioleta Takuska : Exercise 5.3</title>
+    <meta charset="utf-8" />
+
+    <script type="text/javascript">
+      var totalPrice = 0;
+
+      function addItem(cost, elementId) {
+        console.log(elementId)
```

Comment:
I don't think I took points off for this, but I don't like the console logs. Try using the debugger, it'll save you far more time in the long run.